model.py

```diff
@@ -647,6 +647,7 @@ class DataSource(Base):
     PLYMPTON = "Plympton"
     OA_CONTENT_SERVER = "Library Simplified Open Access Content Server"
     PRESENTATION_EDITION = "Presentation edition generator"
+    INTERNAL_PROCESSING = "Library Simplified Internal Process"
```

Comment:
My original idea was to leave DataSource blank for internal processes, but making it explicit is probably better.